cmd/version.go

```diff
@@ -20,11 +26,59 @@ var versionCmd = &cobra.Command{
 To see if there is a more recent version available, call the command with the
 --latest flag.
 	`,
+
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(Version)
+		fmt.Println(currentVersion())
+
+		if checkLatest {
+			c := cli.New(Version)
+			l, err := checkForUpdate(c)
+			if err != nil {
+				Bail(err)
+			}
+			fmt.Println(l)
+		}
+
+		return
 	},
 }
 
+// currentVersion returns a formatted version string for the Exercism CLI.
+func currentVersion() string {
+	cmdName := RootCmd.Use
+	return fmt.Sprintf("%s version %s", cmdName, Version)
+}
+
+// checkForUpdate returns a string indicating if the current running version of the Exercism CLI is at the latest release.
+// For CLI versions found to be out of date a string with upgrade instructions is included in the return.
+func checkForUpdate(c *cli.CLI) (string, error) {
+
+	ok, err := c.IsUpToDate()
+	if err != nil {
+		return "", err
+	}
+
+	o := "Your CLI version is up to date."
+	if ok {
+		return o, nil
+	}
+
+	// If not an exact match let's check if we are over or under
+	cv, _ := semver.Make(c.Version)
+	rv, _ := semver.Make(c.LatestRelease.Version())
```

Comment:
If `IsUpToDate` says "no" if you're over, then lets change `IsUpToDate` so it only complains if you're under.